setup.py

```diff
@@ -1,11 +1,19 @@
 from setuptools import setup, find_packages
+from pathlib import Path
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
 
 setup(
     name='panoptes_client',
     url='https://github.com/zooniverse/panoptes-python-client',
     author='Adam McMaster',
     author_email='adam@zooniverse.org',
```

Comment:
While it is absolutely still the case that Adam is the primary author of this package, I think it is appropriate to point to a Zooniverse email address instead of Adam's old Zoo email address in case of questions or people reaching out.

```suggestion
    author='Adam McMaster / Zooniverse',
    author_email='contact@zooniverse.org',
```